.kiro/steering/triage.md

```
@@ -0,0 +1,193 @@
+```bash
+gh issue edit {number} --remove-label "needs-triage"
+```
+
+## Triage Decision Tree
+
+
+## RealmWalker-Specific Triage
+
+### Known Issue Categories
+
+#### Loom Framework Issues
```

markdownlint-cli2 (0.18.1)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)